apis/python/src/tiledbsoma/_multiscale_image.py

```diff
@@ -93,9 +153,109 @@ def create(
         Lifecycle:
             Experimental.
         """
+        # Warn about the experimental nature of the spatial classes.
         warnings.warn(SPATIAL_DISCLAIMER)
-        raise NotImplementedError()
 
+        context = _validate_soma_tiledb_context(context)
+        if len(set(axis_types)) != len(axis_types):
+            raise ValueError(
+                "Invalid axis types {axis_types} - cannot have repeated values."
+            )
+        if len(axis_names) != len(axis_types):
+            raise ValueError("Mismatched lengths for axis names and types.")
+        axis_type_map = {"channel": "C", "height": "Y", "width": "X", "depth": "Z"}
+        image_type = ""
+        for val in axis_types:
+            try:
+                image_type += axis_type_map[val]
+            except KeyError as ke:
+                raise ValueError("Invalid axis type name '{val}'.") from ke
+        schema = MultiscaleImageSchema(
+            ImageProperties(
+                name="reference_level",
+                image_type=image_type,
```

Comment:
nit: This avoids creating a new immutable string object each time the string is appended

```suggestion
        image_type = []
        for val in axis_types:
            try:
                image_type.append(axis_type_map[val])
            except KeyError as ke:
                raise ValueError("Invalid axis type name '{val}'.") from ke
        schema = MultiscaleImageSchema(
            ImageProperties(
                name="reference_level",
                image_type="".join(image_type),
```

Reply:
Nice.